### ISSUE TITLE
Remove Background Textures

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#10919]
## What
We have a document type in Prismic called background-textures, that allows us to upload images which can then be selected as background in page headers.

It looks like these are intended to be used for events and event-series. Both these custom types have a content relationship field for pulling in one of these background-textures.

However, these don't display for either of these pages and it doesn't look like they have for some time.
- the event-series fetcher doesn't include the the necessary fetchlinks to return the data (although it does look for it in the transformer)
- the event fetcher fetches the necessary data, but doesn't do anything with it in the transformer.

As part of the work for #10781, I've commented out the transformer code which was causing typing errors, rather than trying to fix this as part of that work.

~We need to decide if we need this functionality and, if not,~
Since this functionality has been unavailable for some time and not missed, we should remove it:
- disable the background-textures type in Prismic, 
- remove the content relationship fields from events and event-series
- remove the fetchLinks and related types from the codebase
- remove the commented out code for transforming background-textures data

~If we want to have this functionality, we need to fix it:~
- ~fetch the data for the event-series~
- ~transform the data for events~
- ~make sure we're rendering it on the page~

**Comment by rcantin-w** [comment]
@gestchild I feel like we might as well just kill it if no one's asked for it?

**Comment by gestchild** [comment]
Yeah, I think we should - shall I just alter this ticket to be remove Background Textures?

**Comment by rcantin-w** [comment]
I vote yes!

**Comment by rcantin-w** [comment]
<img width="492" height="879" alt="Image" src="https://github.com/user-attachments/assets/8d607c99-04b1-40be-85e1-519833a580e1" />

destroyed (+ all changes in prod)
